src/components/Profile/Profile.js

```diff
@@ -101,19 +191,27 @@ class UserProfile extends Component {
                     onChange={this.onChange}
                   />
                 </Form.Item>
-                <Form.Item label="New Password">
+                <Form.Item 
+                  label="New Password"
+                  validateStatus={pwdError ? "error" : null}
+                  help={pwdError ? pwdError : null}
+                >
                   <Input
                     type="password"
-                    name="password"
-                    value={password}
+                    name="newpassword"
+                    value={newpassword}
                     onChange={this.onChange}
                   />
                 </Form.Item>
-                <Form.Item label="Confirm New Password">
+                <Form.Item 
+                  label="Confirm New Password"
+                  validateStatus={confPwdError ? "error" : null}
+                  help={confPwdError ? confPwdError : null}
```

Comment:
Is this error displayed in red colour? Screenshot.

Reply:
Yes.
![Uploading pwd.png…]()

Reply:
Okay